src/Sulu/Bundle/CategoryBundle/Exception/CategoryNotFoundException.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\CategoryBundle\Exception;
+
+class CategoryNotFoundException extends \Exception
+{
+    /**
+     * @var mixed
+     */
+    private $categoryIdentifier;
```

Comment:
I changed this to Identifier, because the exception is also thrown if a category for a specific key is not found (CategoryManager::findByKey). Should we use another exception for that case?